llvm/include/llvm/CodeGen/SwitchLoweringUtils.h

```diff
@@ -137,29 +137,34 @@ struct CaseBlock {
   SDLoc DL;
   DebugLoc DbgLoc;
 
-  // Branch weights.
+  // Branch weights and predictability.
   BranchProbability TrueProb, FalseProb;
+  bool IsUnpredictable;
```

Comment:
Where this get used?

Reply:
+1

Reply:
Never mind. I found https://github.com/llvm/llvm-project/commit/09515f2c20111628ce81ad5f40e12e5f6af5ed2f in your description.

Reply:
See `SelectionDAGBuilder::visitSwitchCase`.